parser/parser.y

```diff
@@ -790,7 +792,7 @@ import (
 %left 	'*' '/' '%' div mod
 %left 	'^'
 %left 	'~' neg
-%right 	not
+%right 	not not2
```

Comment:
can we change `not2` to another more specified name ?

Reply:
@zz-jason  MySQL defines token like this. I think it's better to keep consistent with MySQL.

Reply:
ok